packages/base/src/form/form.tsx

```
@@ -24,11 +34,15 @@ const Form = <V extends ObjectType>(props: FormProps<V>) => {
-  const validate = usePersistFn(() => {
-    return formFunc.validateFields();
+  const validate = usePersistFn((fields?: keyof V | Array<keyof V>) => {
+    return formFunc.validateFields(fields as string | string[])
+    .catch((error: ValidationError<V>) => {
+      // 这里可以选择处理错误，或者直接抛出
+      throw error;
+    });;
   });
-  const validateFields = usePersistFn((fileds: string | string[]) => {
-    return formFunc.validateFields(fileds).catch(() => {});
+  const validateFields = usePersistFn((fields: keyof V | Array<keyof V>) => {
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)